ui/perfherder/compare/CompareTableControls.jsx

```diff
@@ -133,6 +142,43 @@ export default class CompareTableControls extends React.Component {
     this.toggleRetriggerModal();
   };
 
+  updateUrlParams = () => {
+    const { updateParams } = this.props.validated;
+    const {
+      filteredText,
+      hideUncomparable,
+      showImportant,
+      hideUncertain,
+      showNoise,
+    } = this.state;
+    const {
+      showOnlyComparable,
+      showOnlyImportant,
+      showOnlyConfident,
+      showOnlyNoise,
+      filter,
+    } = parseQueryParams(window.location.search);
```

Comment:
I think we have all these flags in `this.props.validated`.

Reply:
I've changed the implementation so that it doesn't need the URL params anymore.